### DISCUSSION TITLE
DHCP client for VIP

### DISCUSSION THREAD

**Discussion by NBogos** [acassen/keepalived#2439] [Q&A] [answered]
Hello. Is there a way to set DHCP client for virtual_ipaddress and share it between Peers ? Wanna use vrrp on WAN side but my ISP uses DHCP for authentication/authorization.

**Comment by pqarmitage** [comment, marked as answer]
If I understand your question correctly the answer is no. The virtual_ipaddress(es) must be manually configured in the VRRP configuration.

If you have two peers, Peer A and Peer B, and suppose Peer A is allocated 10.0.0.2 by the DHCP server and Peer B is allocated 10.0.0.3, how would Peer B know that it is supposed to use 10.0.0.2 for the virtual_ipaddress? But perhaps I haven't understood exactly what you want to do.

**Reply by NBogos** [reply]
Yes, you understood correctly
